### PR TITLE
devel: add colorama package

### PR DESCRIPTION
**Pull request by mateusz-bloch**
JIRA CI-296

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 ## NEXT VERSION
 
-## 20230522 (latest)
+## 20230611 (latest)
+* **GCC version: 9.3.0**
+* devel: add colorama package
+* phoenix-rtos-project commit
+  * devel image: `7496f4a`
+  * build image: `e8b8ba5`
```
Comment (damianloew) [inline CHANGELOG.md:10]: I'd not split it and just write `phoenix-rtos-project commit: e8b8ba5`, which relates to a version of project, where build image have been built. @nalajcie do you approve such approach? A version doesn't matter for `devel` image in my opinion.

**Review by damianloew — COMMENTED** [review]
As it's described on Confluence we should firstly commit to this repository and then push images, so we are going to merge it. The images were tested both on Ubuntu host and Rpi. For me looks good, above this one suggestion:

File: CHANGELOG.md (hunk 1 above)
Comment (nalajcie) [reply CHANGELOG.md:10]: It matters, we have `hostutils` in **devel** image version (the `libphoenix`/`toolchain` is in **build** i image version

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
 
 ## NEXT VERSION
 
-## 20230522 (latest)
+## 20230611 (latest)
```
Comment (nalajcie) [inline CHANGELOG.md:5]: invalid date

**Comment by nalajcie** [comment]
> As it's described on Confluence we should firstly commit to this repository and then push images, so we are going to merge it. The images were tested both on Ubuntu host and Rpi.

The intent was to create local commit before building the image (no need to push it upstream before testing), so to do it nicely:
 - this PR should add info in changelog in `NEXT VERSION` section
 - after the PR merge the person which would be building the image, checks out master, creates (local) release commit (new section in chagelog), builds & tests images, and do all of the other necessary work

(if the above steps are being done by the same person in the same day, the first input in the changelog is often omitted in favor of writing the changes only in the second one)

File: CHANGELOG.md (hunk 1 above)
Comment (damianloew) [reply CHANGELOG.md:10]: Yeah that's right, it's worth it. I meant it does not matter for this specific change (adding colorama)

**Review by damianloew — APPROVED** [review]

**Pull request merged**